### PR TITLE
Create docs for Playwright Test (Beta)

### PR DESCRIPTION
**Pull request by darkostanimirovic**
## Affected Components
* [ ] Content & Marketing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

## Notes for the Reviewer
- Still a draft, I haven't finished all the pieces.
- I want to look into good images or video but not sure I'll get to that this week.
- Since this is a Public Beta, it's OK that it's a little simplistic. It will still give enough information, especially the list of supported/unsupported features.

### CODE REVIEW

**Comment by darkostanimirovic** [comment]
@checklyalex Can you take a look now? I'm a little unsure about the GIFs. Do we have like a standard for how these should look? Eventually these should probably be actual YT videos right?

File: site/content/docs/browser-checks/playwright-test.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+Checkly natively supports running browser checks using Playwright Test Runner, allowing you to write tests and use assertions using the popular testing framework.
+
+{{<info >}}
+
+Support for Playwright Test is currently in Public Beta. If you experience an issue or have suggestions, write to us at support@checklyhq.com or post to our Public Roadmap.
+
+{{</info >}}
+
+## Getting started
+Playwright test is currently only available on the latest runtime, 2022.02.
```
Comment (checklyalex) [inline site/content/docs/browser-checks/playwright-test.md:19]: Let's make sure 'Playwright Test' is always capitalised.

File: site/content/docs/browser-checks/playwright-test.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+You can download the trace files for manual inspection or view them directly with [trace.playwright.dev](https://trace.playwright.dev).
+
+![checkly-pw-traces](https://user-images.githubusercontent.com/3121310/194862081-4c5ef0db-84fa-4ac5-b63c-90853ba748a6.gif)
+
+Note: At the moment, traces are only recorded for failing checks.
+
+### Video recordings
+When a browser check that uses Playwright Test fails, it will record a video for each page navigation and make it available in the UI.
+
+Here's an example of a Playwright test script that fails, and provides a video of the test sequence.
```
Comment (checklyalex) [inline site/content/docs/browser-checks/playwright-test.md:78]: Same again here

File: site/content/docs/browser-checks/playwright-test.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+{{<info >}}
+
+Support for Playwright Test is currently in Public Beta. If you experience an issue or have suggestions, write to us at support@checklyhq.com or post to our Public Roadmap.
+
+{{</info >}}
+
+## Getting started
+Playwright test is currently only available on the latest runtime, 2022.02.
+
+All it takes from there is to create a new browser check and write a Playwright Test script:
```
Comment (checklyalex) [inline site/content/docs/browser-checks/playwright-test.md:21]:
```suggestion
A simple Playwright Test script would look like this:
```

File: site/content/docs/browser-checks/playwright-test.md (hunk 2 above)
Comment (checklyalex) [inline site/content/docs/browser-checks/playwright-test.md:78]:
```suggestion
Here's an example of a Playwright Test script that fails, and provides a video of the test sequence.
```

File: site/content/docs/browser-checks/playwright-test.md (hunk 1 above)
Comment (checklyalex) [inline site/content/docs/browser-checks/playwright-test.md:19]:
```suggestion
Playwright Test is currently only available on the latest runtime, 2022.02.
```

**Review by checklyalex — CHANGES_REQUESTED** [review]
Looks good enough to get us started. We need to strip out of the purple from the videos to [keep the look and feel consistent](https://www.notion.so/checkly/WIP-Docs-Style-Guide-69f5792000464e398b68a45c3fb4042a#db82436457a24cd09161e7a328dda205). If you can do that before tomorrow, it would be great. If not, we can get this out.

**Comment by darkostanimirovic** [comment]
@checklyalex I've updated the GIFs, can you take a look now? I used the Guest mode.

**Review by checklyalex — APPROVED** [review]
LGTM

**Pull request merged**